### DISCUSSION TITLE
Generate a sitemap during build/generate?

### DISCUSSION THREAD

**Discussion by SebbeJohansson** [nuxt/nuxt#16384] [Questions] [answered]
Hi! @danielroe said he would write an article on how to generate a sitemap during build/generate for dynamic routes. I cant find any such article or anything in the docs. I tried to use https://github.com/benoitdemaegdt/nuxt3-sitemap, but I couldn't get it to work.

Does anyone know of any articles or guides on how to do this?

**Comment by IsraelOrtuno** [comment, marked as answer]
@SebbeJohansson this may be the article by @danielroe you are looking for:
https://roe.dev/blog/creating-your-own-sitemap-module

This guide from the content module itself may be also helpful:
https://content.nuxtjs.org/guide/recipes/sitemap

**Reply by SebbeJohansson** [reply]
@IsraelOrtuno Yes that would be the article i was waiting for. That guide is new!
